src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -209,7 +209,7 @@ static List * GreedyAssignTaskList(List *taskList);
 static Task * GreedyAssignTask(WorkerNode *workerNode, List *taskList,
 							   List *activeShardPlacementLists);
 static List * ReorderAndAssignTaskList(List *taskList,
-									   List * (*reorderFunction)(Task *, List *));
+									   List * (*reorderFunction)(List *));
```

Comment:
while you are removing the unused parameter, can you also use a typedef for these reoderFunctions? 

something like:

```c
typedef List* (*ReorderFunction)(List *);
```

then we can use it like :

```c
ReorderAndAssignTaskList(List* taskList, ReorderFunction reorderFunction)
```